GrpcDotNetNamedPipes/NamedPipeServerOptions.cs

```diff
@@ -31,4 +31,17 @@ public class NamedPipeServerOptions
     /// </summary>
     public PipeSecurity PipeSecurity { get; set; }
 #endif
+
+    /// <summary>
+    /// Gets or sets a Custom Task Factory to control how tasks are serviced. 
+    /// For example, causing threads to be processsed in FIFO rather than LIFO  
+    /// by using TaskCreationOptions.preferFairness
+    /// </summary>
+    public TaskFactory TaskFactory { get; set; }
+
+    /// <summary>
+    /// Gets or sets a count of threads to use for the listener.
+    /// If you need to address a synchronous code execution issue, try increasing
+    /// </summary>
+    public int ThreadPoolSize { get; set; } = 4;
```

Comment:
Are you sure this helps? In general the listener thread pool is sitting idle and whenever a connection is established, immediately hands off to a task on a different thread and goes back to idling. You shouldn't need a ton of threads to be able to handle that.

If there is a problem it's presumably in high-contention scenarios where clients have trouble connecting, but in that scenario I would rather make improvements to client connection reliability that work out of the box rather than adding a knob that needs to be tuned.